addon/components/md-btn-dropdown.js

```diff
@@ -22,12 +22,12 @@ export default MaterializeButton.extend({
 
     var that = this;
     this.$().dropdown({
-      hover: this.getWithDefault('hover', false) === 'true',
-      constrainWidth: this.getWithDefault('constrainWidth', true) === 'true',
+      hover: this.getWithDefault('hover', false),
```

Comment:
I don't know what `$.dropdown` expects, but to be save we should cast truthy values to boolean values

For example:
```js
hover: !!this.getWithDefault('hover', false),
```